ngx_http_vhost_traffic_status_string.c:182:12: error: expected '(' after 'if'         if isascii(*pa) {

/Users/lee/Developer/nginx/modules/nginx-module-vts/src/ngx_http_vhost_traffic_status_string.c:182:12: error: expected '(' after 'if'
        if isascii(*pa) {
           ^
1 error generated.
make[2]: *** [objs/addon/src/ngx_http_vhost_traffic_status_string.o] Error 1
make[1]: *** [build] Error 2
make: *** [all] Error 2

--- a/src/ngx_http_vhost_traffic_status_string.c
+++ b/src/ngx_http_vhost_traffic_status_string.c
@@ -179,7 +179,7 @@ ngx_http_vhost_traffic_status_escape_prometheus(ngx_pool_t *pool, ngx_str_t *buf
 
     /* Find the first character that needs to be escaped */
     while (pa < last) {
-        if isascii(*pa) {
+        if (isascii(*pa)) {
             if (*pa == '"' || *pa == '\\' || *pa == '\n') {
                 break;
             } else {

